src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/MemoryFormStore.js

```diff
@@ -47,7 +47,7 @@ export default class MemoryFormStore extends AbstractFormStore implements FormSt
     }
 
     @action setMultiple(data: Object) {
-        this.data = data;
+        this.data = {...this.data, ...data};
```

Comment:
Actually this is quite a change in the semantics... Would it at leaset be consistent with the behavior of the `ResourceFormStore`?

Reply:
Yes.

https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/ResourceFormStore.js#L155

https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceStore/ResourceStore.js#L268